Installed in HEAD.  Needed for glibc-2_7-branch.


2007-09-13  H.J. Lu  <hongjiu.lu@intel.com>

	* sysdeps/i386/i586/memcpy_chk.S: New file.
	* sysdeps/i386/i586/mempcpy_chk.S: Likewise.
	* sysdeps/i386/i586/memset_chk.S: Likewise.

diff --git a/sysdeps/i386/i586/memcpy_chk.S b/sysdeps/i386/i586/memcpy_chk.S
new file mode 100644
index 0000000..ab8a95c
--- /dev/null
+++ b/sysdeps/i386/i586/memcpy_chk.S
@@ -0,0 +1 @@
+#include <sysdeps/i386/i686/memcpy_chk.S>
diff --git a/sysdeps/i386/i586/mempcpy_chk.S b/sysdeps/i386/i586/mempcpy_chk.S
new file mode 100644
index 0000000..9a1de1d
--- /dev/null
+++ b/sysdeps/i386/i586/mempcpy_chk.S
@@ -0,0 +1 @@
+#include <sysdeps/i386/i686/mempcpy_chk.S>
diff --git a/sysdeps/i386/i586/memset_chk.S b/sysdeps/i386/i586/memset_chk.S
new file mode 100644
index 0000000..09f9d42
--- /dev/null
+++ b/sysdeps/i386/i586/memset_chk.S
@@ -0,0 +1 @@
+#include <sysdeps/i386/i686/memset_chk.S>
